api-spec/protobuf/ark/v1/service.proto

```diff
@@ -246,9 +247,8 @@ message RoundFailed {
 
 message RoundSigningEvent {
   string id = 1;
-  repeated string cosigners_pubkeys = 2;
-  Tree unsigned_vtxo_tree = 3;
-  string unsigned_round_tx = 4;
+  Tree unsigned_vtxo_tree = 2;
```

Comment:
let's keep the list of cosigners, this way it's easier for clients to understand if they are participating this round or have to wait for the next one.